docs/docs-beta/docs/guides/pipelines.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Create a pipeline"
+description: "Learn how to create data pipelines using Dagster's asset-based approach"
+---
+
+In Dagster, data pipelines are created using an asset-based approach. This overview will introduce you to the key concepts and steps involved in building a Dagster pipeline.
+
+## What you'll learn
```

Comment:
Take this out, given that you're doing it elsewhere?

Reply:
was removed in the upstream PR